Model/Data/Email.php

```diff
@@ -0,0 +1,136 @@
+<?php
+namespace Omise\Payment\Model\Data;
+
+class Email
+{
+    /**
+    * @var \Magento\Framework\App\Config\ScopeConfigInterface
+    */
+    protected $_scopeConfig;
+
+    /**
+     * @var \Omise\Payment\Model\Api\Charge
+     */
+    protected $_charge;
+
+    /**
+     * @var \Magento\Framework\View\Asset\Repository
+     */
+    protected $_assetRepo;
+
+    /**
+    * @var  \Magento\Framework\Mail\Template\TransportBuilder
+    */
+    private $_transportBuilder;
+
+    /**
+     * @var \Magento\Store\Model\StoreManagerInterface
+     */
+    protected $_storeManager;
+
+    /**
+     * @var \Omise\Payment\Helper\OmiseHelper
+     */
+    protected $_helper;
+
+    /**
+     * @var string
+     */
+    private $storeName;
+
+    /**
+     * @var string
+     */
+    private $emailTemplate;
+    
+
+    public function __construct(
+        \Magento\Framework\App\Config\ScopeConfigInterface $scopeConfig,
+        \Omise\Payment\Model\Api\Charge  $charge,
+        \Magento\Framework\View\Asset\Repository $assetRepo,
+        \Magento\Framework\Mail\Template\TransportBuilder $transportBuilder,
+        \Magento\Store\Model\StoreManagerInterface $storeManager,
+        \Omise\Payment\Helper\OmiseHelper $helper
+    ) {
+        $this->_scopeConfig = $scopeConfig;
+        $this->_charge      = $charge;
+        $this->_assetRepo = $assetRepo;
+        $this->_transportBuilder = $transportBuilder;
+        $this->_storeManager = $storeManager;
+        $this->_helper           = $helper;
+    }
```

Comment:
Maybe need to clean up the code alignment a bit? (either vertical align or 1 space)

Reply:
done @ 81b078e0b967d16b53bfb532673fd627c51c857f